pnpm-lock.yaml

```diff
@@ -1,119 +1,98 @@
-lockfileVersion: '6.0'
+lockfileVersion: 5.4
```

Comment:
We need to address this. We cannot wage lock file wars in ever PR. Looking into this now...